test/e2e/quadlet/stopsignal.container

```diff
@@ -0,0 +1,5 @@
+## assert-podman-args "--stop-signal" "SIGKILL"
+
+[Container]
+Image=localhost/imagename
+StopTimeout=SIGKILL
```

Comment:
s/StopTimeout/StopSignal

Reply:
Yes, this is what causes the tests to fail

Reply:
Always a cut and paste error bites  you.